MobileCenterPush/MobileCenterPush/MSPush.m

```diff
@@ -199,17 +199,14 @@ - (void)channel:(id<MSChannel>)channel didFailSendingLog:(id<MSLog>)log withErro
 
 #pragma mark - Register callbacks
 
-- (void)application:(UIApplication *)application
-    didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
-  (void)application;
+- (void)didRegisterForRemoteNotificationsWith:(NSData *)pushToken {
   MSLogVerbose([MSPush logTag], @"Registering for push notifications has been finished successfully");
-  NSString *strDeviceToken = [self convertTokenToString:deviceToken];
-  [MS_USER_DEFAULTS setObject:strDeviceToken forKey:kMSPushServiceStorageKey];
-  [self sendDeviceToken:strDeviceToken];
+  NSString *strPushToken = [self convertTokenToString:pushToken];
+  [MS_USER_DEFAULTS setObject:strPushToken forKey:kMSPushServiceStorageKey];
+  [self sendPushToken:strPushToken];
 }
 
-- (void)application:(UIApplication *)application didFailToRegisterForRemoteNotificationsWithError:(NSError *)error {
-  (void)application;
+- (void)didFailToRegisterForRemoteNotificationsWith:(NSError *)error {
```

Comment:
I am not sure about the naming of the proxied delegate methods, we'll have to discuss about it with PMs probably. But I would still rename this one to `didFailToRegisterForRemoteNotificationsWithError:`.